.github/workflows/extra.yml

```diff
@@ -67,9 +70,10 @@ jobs:
           selected = "${{ matrix.extras_to_install }}".split()
           WANT = {
             "data": ["pandas","dateutil"],
+            "json": ["sdmxschemas", "jsonschema"],
             "xml":  ["lxml","sdmxschemas","xmltodict"],
             "xml-data":  ["lxml","sdmxschemas","xmltodict","pandas","dateutil"],
-            "vtl":  ["vtlengine","pandas","dateutil"],
+            "vtl":  ["vtlengine","pandas","dateutil", "jsonschema"],
```

Comment:
I'm sure it's correct but, just out of curiosity: For the vtl extra, we need a dependency on jsonschema, but not on sdmxschemas and not on XML?